crm-plugin/controllers/tag.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+/**
+ * crm-plugin.js controller
+ *
+ * @description: A set of functions called "actions" of the `crm-plugin` plugin.
+ */
+const { sanitizeEntity } = require("strapi-utils");
+module.exports = {
+  /**
+   * Default action.
+   *
+   * @return {Object}
+   */
+
+  index: async (ctx) => {
+    // Add your own logic here.
+
+    // Send 200 `ok`
+    ctx.send({
+      message: "ok",
+    });
+  },
+
+  find: async (ctx) => {
+    let tag = await strapi.query("tag", "crm-plugin").find(ctx.query);
+    return tag.map((entity) =>
+      sanitizeEntity(entity, {
+        model: strapi.plugins["crm-plugin"].models["tag"],
+      })
+    );
+  },
+
+  create: async (ctx) => {
+    let entity;
+    let contacttagEntry;
+    try {
+      if (ctx.params.id) {
+        const { id } = ctx.params;
+        entity = await strapi
```

Comment:
order is wrong.. relations table should be updated first and then the main table